stubs/pyaudio/METADATA.toml

```diff
@@ -1,7 +1,6 @@
 version = "0.2.*"
-# There is no web portal for the source, this is the official source link:
-# https://people.csail.mit.edu/hubert/pyaudio/#sources
-upstream_repository = "https://people.csail.mit.edu/hubert/git/pyaudio.git"
+# There is no web portal for the source, this is the official source link
+upstream_repository = "https://people.csail.mit.edu/hubert/pyaudio/#sources"
```

Comment:
@AlexWaygood This is my usage of a git URL or a fragment. I'll let you decide on whether this value or the restrictive parsing should change, depending on your needs. It's definitely an edge case.

Reply:
I've updated the `upstream_repository` field validation. I believe this respects the original intent of avoiding extra query params and fragments that may have accidentally been copied.